### PR TITLE
Remove /invis smite, add /invis clear

### LINKED ISSUE #959: Remove smiting all invisible players
Hi,
As clarified, Mark says invisible potions are allowed. See http://totalfreedom.boards.net/post/81230/. As a result, we should remove '/invis smite'

### PR DESCRIPTION
**Pull request by LegendIsAwesomes**
Resolves #959

### CODE REVIEW

**Comment by Wild1145** [comment]
Again, as stated on the issue I disagree with removing TFM functionality for the sake of it.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_invis.java [hunk 1]
```diff
@@ -11,19 +11,19 @@
 import org.bukkit.potion.PotionEffectType;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
-@CommandParameters(description = "Shows (optionally smites) invisisible players", usage = "/<command> (smite)")
+@CommandParameters(description = "Shows (optionally clears) invisisible players", usage = "/<command> (clear)")
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_invis.java]: This should be an optional parameter. Usage is `[parameter]`

**Comment by JeromSar** [comment]
@LegendIsAwesomes One small change and this PR is good to go.

**Comment by LegendIsAwesomes** [comment]
There we go @JeromSar

**Comment by JeromSar** [comment]
@LegendIsAwesomes Thanks for contributing! This PR is ready to merge and will be soon. :)

**Comment by DUFCLiam** [comment]
No! Add it dont remove `/invis smite` please!

**Comment by DUFCLiam** [comment]
@JeromSar

**Comment by LegendIsAwesomes** [comment]
It's been accepted, invis smite just smites innocents

**Comment by JeromSar** [comment]
@DUFCLiam The implementation is fine as-is. For reasoning behind removing /invis smite, see the original issue.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_invis.java [hunk 2]
```diff
@@ -53,9 +53,9 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
             return true;
         }
 
-        if (smite)
+        if (clear)
         {
-            playerMsg("Smitten " + smites + " players");
+            playerMsg("Cleared invisibility effect from " + clears + " players"");
```
Comment (OnlineRepz) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_invis.java]: You added 2 "'s at the end.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_invis.java [hunk 3]
```diff
@@ -11,19 +11,19 @@
 import org.bukkit.potion.PotionEffectType;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
-@CommandParameters(description = "Shows (optionally smites) invisisible players", usage = "/<command> (smite)")
+@CommandParameters(description = "Shows (optionally clears) invisisible players", usage = "/<command> [clear]")
```
Comment (OnlineRepz) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_invis.java]: Spelling typo.

Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_invis.java]: @LegendIsAwesomes May be something to fix while you're at it. :)

**Comment by LegendIsAwesomes** [comment]
Fixed. @JeromSar

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_invis.java (hunk 3 above)
Comment (AvalancheYT) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_invis.java]: @LegendIsAwesomes

**Review by Wild1145 — APPROVED** [review]
Should be ready to merge.

**Comment by JeromSar** [comment]
Merged in https://github.com/TotalFreedom/TotalFreedomMod/commit/0eb0c7a02f040f9347369518f94bcc1173cb719d. Thanks for contributing!

**Pull request closed without merging**